Documentation/building-and-flashing-firmware.md

```diff
@@ -35,6 +35,42 @@ Select BusPirate_v3 or BusPirate_v4 from the dropdown menu in the toolbar
 
 Click the hammer in the toolbar.
 
-Your firmware will be saved in Bus_Pirate/Firmware/busPirate.X/dist/BusPirate_v*/production/busPirate.X.production.hex
+The firmware will be saved in Bus_Pirate/Firmware/busPirate.X/dist/BusPirate_v[3 or 4]/production/busPirate.X.production.hex
 
 # Flashing
+
+To flash the firmware onto a Bus Pirate, there is a tool called Pirate-Loader. This tool is included in the Bus Pirate Github repository, so there's nothing extra to download. 
+
+Simply navigate to Bus_Pirate/package/BPv3-firmware. Here you will find pirate-loader_lnx, pirate-loader_mac and pirate-loader.exe, for the respective platforms. When you see `pirate-loader` in a command in this section replace it with either `./pirate-loader_lnx` `./pirate-loader_mac` or `pirate-loader.exe`, depending on your platform. You may wish to copy pirate-loader to another location or add it to your PATH environment variable. 
```

Comment:
This is probably going to change real soon, as older packages will be removed.  The current "official" procedure is to rebuild your own pirate-loader from /Bootloaders/BPv3-bootloader/pirate-loader, then use CMake to build the binary.  Nothing to change in the documentation right now, just a heads up :)